Lib/test/test_zipfile.py

```diff
@@ -611,8 +611,13 @@ def test_add_file_before_1980(self):
 
     def test_add_file_after_2107(self):
         # Set atime and mtime to 2108-12-30
+        ts = 4386268800
         try:
-            os.utime(TESTFN, (4386268800, 4386268800))
+            time.localtime(ts)
+        except OverflowError:
+            self.skipTest(f'time.localtime({ts}) raises OverflowError')
```

Comment:
Please rebase your PR.

Reply:
rebase is, perhaps, not needed.

aixtools@x064:[/data/prj/python/git/python3-3.9]git rebase master
First, rewinding head to replay your work on top of it...
Applying: Fix PyTime_localtime() on AIX so that only 32-bit is range
tested for overflow
Applying: modify macro format and variable tested
Applying: Modify blurb to specify user function name, rather than
internal function name
Applying: merge, manually, PR18282 into `Lib/test/test_zipfile.py`
Using index info to reconstruct a base tree...
M       Lib/test/test_zipfile.py
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.

So, are we good to go? Or am I missing something?

Reply:
"ts = 4386268800" is already in master. I don't understand what's wrong with your PR. Maybe try to write a new PR on top of up to date master.

Reply:
force-pushed a PR based on today's master.